dotCMS/src/main/java/com/dotcms/publisher/bundle/business/BundleAPI.java

```diff
@@ -111,6 +115,43 @@
 	 */
 	public void deleteBundle(String id) throws DotDataException;
 
+	/**
+	 * Deletes a bundle with given id, plus all dependencies such as asset in the queue, audit, etc.
+	 *
+	 * Assets will be removed by cron job
+	 * @param id {@link String}
+	 * @param user {@link com.liferay.portal.model.User} user to use to check permissions
+	 * @throws DotDataException
+	 */
+	public void deleteBundleAndDependencies(String id, User user) throws DotDataException;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary modifier 'public' on method 'deleteBundleAndDependencies': the method is declared in an interface type](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4502981)